src/modules/live-execution-tracker/repositories/live-execution-tracker.repository.ts

```diff
@@ -21,6 +21,10 @@ export class LiveExecutionTrackerRepository implements ILiveExecutionTrackerRepo
     this._managementApiClient = managementApiClientService;
   }
 
+  public async getFlowNodeInstancesForProcessInstance(processInstanceId: string): Promise<Array<DataModels.FlowNodes.FlowNodeInstance>> {
```

Comment:
With regards to my remarks [here](https://github.com/process-engine/management_api_contracts/pull/50#discussion_r294163731)

```suggestion
  public async getFlowNodeInstancesForProcessInstance(processInstanceId: string): Promise<Array<DataModels.FlowNodeInstances.FlowNodeInstance>> {
```